conans/test/unittests/model/editable_cpp_info/parse_test.py

```diff
@@ -5,96 +5,32 @@
 import shutil
 import unittest
 
-import six
-
+from conans.errors import ConanException
 from conans.model.editable_cpp_info import EditableCppInfo
 from conans.test.utils.test_files import temp_folder
 from conans.util.files import save
 
-base_content = six.u(textwrap.dedent("""\
-    [{namespace}includedirs]
-    {path_prefix}dirs/includedirs
-
-    [{namespace}libdirs]
-    {path_prefix}dirs/libdirs
-
-    [{namespace}resdirs]
-    {path_prefix}dirs/resdirs
-
-    #[{namespace}bindirs]
-    #{path_prefix}dirs/bindirs
-
-    [{namespace}extradirs]
-    {path_prefix}dirs/extradirs
-    """))
-
-
-class NoNamespaceFileTest(unittest.TestCase):
-    content = '\n\n'.join([
-        base_content.format(namespace="", path_prefix=""),
-        base_content.format(namespace="*:", path_prefix="all/"),
-        base_content.format(namespace="libA:", path_prefix="libA/")
-    ])
 
+class ParseTest(unittest.TestCase):
```

Comment:
We need here:
 * parsing a pattern -> fail
 * parsing a full reference with revision -> fail